application/android/app/src/main/java/com/group12/beabee/views/MainStructure/MainActivity.java

```diff
@@ -91,7 +92,7 @@ public Fragment createFragment(int position) {
                     baseContainerFragment = fragmentActivity.containers[3];
                     break;
                 case 4:
-                    fragmentActivity.containers[4] = new BaseContainerFragment(new HomeFragment());
+                    fragmentActivity.containers[4] = new BaseContainerFragment(new FeedFragment());
                     baseContainerFragment = fragmentActivity.containers[4];
                     break;
                 default:
```

Comment:
nice naming  !

Reply:
It's a good choice that you didnt seperate bottom bar into 6 and seperated the feed page into 2.